docs/source/configuration_file.rst

```diff
@@ -761,11 +761,9 @@ Split Dataset
         "$schema": "http://json-schema.org/draft-04/schema#",
         "title": "fname_split",
         "$$description": [
-            "File name of the log (`joblib <https://joblib.readthedocs.io/en/latest/>`__)\n",
-            "that contains the list of training/validation/testing filenames. This file can later\n",
-            "be used to re-train a model using the same data splitting scheme. If ``null``,\n",
-            "a new splitting scheme is performed. If specified, the .joblib file data splitting scheme\n",
-            "bypasses all the other split dataset parameters. Default: ``null``."
+            "Name of the `joblib <https://joblib.readthedocs.io/en/latest/>`__ file that was generated during a previous training, and that contains the list of training/validation/testing filenames.\n",
```

Comment:
```suggestion
            "A `joblib <https://joblib.readthedocs.io/en/latest/>`__ file of the name split_datasets.joblib was generated during a previous training which contains the list of training/validation/testing filenames.\n",
```

Reply:
The way this is rephrase makes it less clear that this refers to a parameter description. 

when it starts with "Name of the `joblib..." it is clear that this is what the param is.

when it starts with "A `joblib file of the name split_datasets.joblib was generated during a previous training..." this reads like an isolated sentence, not what the param actually is